cpp/include/raft/sparse/mst/detail/mst_solver_inl.cuh

```diff
@@ -339,15 +341,16 @@ void MST_solver<vertex_t, edge_t, weight_t>::min_edge_per_supervertex() {
   // the above kernel only adds directed mst edges in the case where
   // a pair of vertices don't pick the same min edge between them
   // so, now we add the reverse edge to make it undirected
-  detail::add_reverse_edge<<<nblocks, nthreads, 0, stream>>>(
-    new_mst_edge_ptr, indices, weights, temp_src_ptr, temp_dst_ptr,
-    temp_weights_ptr, v);
+  //  detail::add_reverse_edge<<<nblocks, nthreads, 0, stream>>>(
```

Comment:
An extra comment here to document why this kernel launch is commented

Reply:
I had forgotten about this change. @afender, this will probably break cugraph if we remove the symmetry altogether right? Should we make this optional?

Reply:
If this is made optional, then the changes to the above additions will also have to be made optional

Reply:
Just to make note here- we talked offline about this. @divyegala will push necessary updates directly to this branch (or open a follow-on if necessary). SLHC is dependent upon this change but we can't break cugraph's assumption of symmetric outputs.